### PR TITLE
quest_template_addon.md changes and corrections

### PR DESCRIPTION
**Pull request by TheSCREWEDSoftware**
Corrected previous error about the Loremaster Description. Fixed the headings of the page that were not previously working, corrected some punctionation and removed links that didn't show the mentioned thing(s).

### Description

- Fixed the headings
- Corrected an error on the 256 Loremaster Descrition
- Corrected Punction
- Removed Links that didn't work / lead to anything within the page
### Related Issue

Closes

## Thank you for contributing to the AzerothCore wiki.

Remember that the wiki is currently available in English and Spanish.

- https://www.azerothcore.org/wiki/home
- https://www.azerothcore.org/wiki/es/home

@pangolp Changes in the EN

### CODE REVIEW

**Comment by TheSCREWEDSoftware** [comment]
Correction of loremaster from: https://github.com/azerothcore/wiki/pull/994

File: docs/quest_template_addon.md [hunk 1]
```diff
@@ -10,41 +10,24 @@ Contains extra definitions like linking quests, dependencies and requirements fo
...
+| [RequiredSkillID](#requiredskillid)       | SMALLINT  | UNSIGNED   |     | NO   |         |       |                                       |
+| [RequiredSkillPoints](#requiredskillpoints)   | SMALLINT  | UNSIGNED   |     | NO   |         |       |                                       |
+| [RequiredMinRepFaction](#requiredminrepfaction) | SMALLINT  | UNSIGNED   |     | NO   |         |       |                                       |
+| [RequiredMaxRepFaction](#requiredmaxrepfaction) | SMALLINT  | UNSIGNED   |     | NO   |         |       |                                       |
+| [RequiredMinRepValue](#requiredminrepvalue)   | MEDIUMINT |            |     | NO   |         |       |                                       |
+| [RequiredMaxRepValue](#requiredmaxrepvalue)   | MEDIUMINT |            |     | NO   |         |       |                                       |
+| [ProvidedItemCount](#provideditemcount)     | TINYINT   | UNSIGNED   |     | NO   |         |       |                                       |
+| [SpecialFlags](#specialflags)          | TINYINT   | UNSIGNED   |     | NO   |         |       |                                       |
```
Comment (Helias) [inline docs/quest_template_addon.md:29]: if you use an auto formatter this text will have a better shape

Comment (TheSCREWEDSoftware) [reply docs/quest_template_addon.md:29]: I'm using Visual Studio Code for ending the md files, is there a specific functionality within it or it's like an extension to it?

Comment (Helias) [reply docs/quest_template_addon.md]: yep

**Pull request merged**